annot-tsv.1

```diff
@@ -0,0 +1,216 @@
+\fB\-\-version\fR
+.RS 4
+Print version string and exit
+.RE
+.PP
+\fB\-a, \-\-annotate\fR \fILIST\fR
+.RS 4
+Add one or more special annotation
+.PP
+\fIcnt\fR
+.RS 4
+number of overlapping regions
+.RE
+.PP
+\fIfrac\fR
+.RS 4
+fraction of the target region with an overlap
+.RE
+.PP
+\fInbp\fR
+.RS 4
+number of source base pairs in the overlap
+.RE
+.RE
+.PP
+\fB\-H, \-\-ignore\-headers\fR
+.RS 4
+Ignore the headers completely and use numeric indexes even when a header exists
+.RE
+.PP
+\fB\-O, \-\-overlap\fR \fIFLOAT\fR
+.RS 4
+Minimum overlap as a fraction of region length in at least one of the overlapping regions\&. If also
+\fB\-r, \-\-reciprocal\fR
+is given, require at least
+\fIFLOAT\fR
+overlap with respect to both regions
+.RE
+.PP
+\fB\-r, \-\-reciprocal\fR
+.RS 4
+Require the
+\fB\-O, \-\-overlap\fR
+with respect to both overlapping regions
+.RE
+.PP
+\fB\-x, \-\-drop\-overlaps\fR
+.RS 4
+Drop overlapping regions (cannot be combined with
+\fB\-f, \-\-transfer\fR)
+.RE
+.SH "EXAMPLE"
+.sp
+Both SRC and TGT input files must be tab\-delimited files with or without a header, their columns can be named differently, can appear in arbitrary order\&. For example consider the source file
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+#chr   beg   end   sample   type   qual
+chr1   100   200   smpl1    DEL    10
+chr1   300   400   smpl2    DUP    30
+.fi
+.if n \{\
+.RE
+.\}
+.sp
+and the target file
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+150   200   chr1   smpl1
+150   200   chr1   smpl2
+350   400   chr1   smpl1
+350   400   chr1   smpl2
+.fi
+.if n \{\
+.RE
+.\}
+.sp
+In the first example we transfer type and quality but only for regions with matching sample\&. Notice that the header is present in SRC but not in TGT, therefore we use column indexes for the latter
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+annot\-tsv \-s src\&.txt\&.gz \-t tgt\&.txt\&.gz \-c chr,beg,end:3,1,2 \-m sample:4 \-f type,qual
+150   200   chr1   smpl1   DEL   10
+150   200   chr1   smpl2   \&.     \&.
+350   400   chr1   smpl1   \&.     \&.
+350   400   chr1   smpl2   DUP   30
+.fi
+.if n \{\
+.RE
+.\}
+.sp
+One of the SRC or TGT file can be streamed from stdin
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+cat src\&.txt | annot\-tsv \-t tgt\&.txt \-c chr,beg,end:3,2,1 \-m sample:4 \-f type,qual \-o output\&.txt
+cat tgt\&.txt | annot\-tsv \-s src\&.txt \-c chr,beg,end:3,2,1 \-m sample:4 \-f type,qual \-o output\&.txt
+.fi
+.if n \{\
+.RE
+.\}
+.sp
+The program can be used in a grep\-like mode to print only matching regions of the target file without modifying the records
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+annot\-tsv \-s src\&.txt \-t tgt\&.txt \-c chr,beg,end:3,2,1 \-m sample:4
+150   200   chr1   smpl1
+350   400   chr1   smpl2
+.fi
+.if n \{\
+.RE
+.\}
+.SH "AUTHORS"
+.sp
+The program was written by Petr Danecek and was originally published on github as annot\-regs
+.SH "COPYING"
+.sp
+The MIT/Expat License or GPL License, see the LICENSE document for details\&. Copyright (c) Genome Research Ltd\&.
```

Comment:
MIT/Expat only.  GPL is not currently mentioned in the LICENSE document, and I don't see why anyone would want to use it if you can go for MIT/Expat instead.